### PR TITLE
List and Manage List styling

### LINKED ISSUE #58: 28. List Styling
The following elements should be styled on our `List` page according to our prototype:
- [x]  `Header`: List name
- [x]  `Label`: Search for an item (rename to Filter?)
- [x] `Input Search Box`: Search for an item by name
- [x] `Button`: Reset
- [x] `Label`: Add new item
- [x] `Input Text Box`: Item Name
- [x] `Input Select Dropdown`: Next Purchase Date
- [x] `Button`: Add item
- [x] `List Items`: Items in the user's list
> - [x] Checkbox
> - [x] Icon
> - [x] Item Name
> - [x] Purchase Urgency
> - [x] `Button`: Delete

### LINKED ISSUE #59: 29. Manage List Styling
The following elements from our `Manage List` page should be styled according to our prototype:

- [x] `Header`: List name
- [ ] `Header`: Manage List
- [x] `Label`: Share List
- [x] `Input Text Box`: Enter Recipient's Email Address
- [x] `Button`: Send Invite!
- [x] `Button`: Delete List

### PR DESCRIPTION
**Pull request by etiry**
## Description

- Styled individual list items, add item form, list name header, share list form, delete list button, and List view conditional components (e.g., messages when list is empty or not selected)

## Related Issue

Closes #58 
Closes #59 

## Acceptance Criteria

The following elements should be styled on our `List` page according to our prototype:
- [x]  `Header`: List name
- [x]  `Label`: Search for an item (rename to Filter?)
- [x] `Input Search Box`: Search for an item by name
- [x] `Button`: Reset
- [x] `Label`: Add new item
- [x] `Input Text Box`: Item Name
- [x] `Input Select Dropdown`: Next Purchase Date
- [x] `Button`: Add item
- [x] `List Items`: Items in the user's list
> - [x] Checkbox
> - [x] Icon
> - [x] Item Name
> - [x] Purchase Urgency
> - [x] `Button`: Delete
- [x] `Button`: Select a list
- [x] `Header`: List is currently empty
- [x] `Header`: You haven't selected a list

The following elements from our `Manage List` page should be styled according to our prototype:

- [x] `Header`: List name
~~- [ ] `Header`: Manage List~~ _(We opted to exclude this design element.)_
- [x] `Label`: Share List
- [x] `Input Text Box`: Enter Recipient's Email Address
- [x] `Button`: Send Invite!
- [x] `Button`: Delete List

## Type of Changes

`enhancement`

### Before

List view - no list selected
image

Manage List view - no list selected
image

List view - empty list
image

List view - populated list
image

Managed List view - list selected
image

### After
List view - no list selected
image

Manage List view - no list selected (should look the same as List view)
image

List view - empty list
image

List view - populated list
image

Manage List view - list selected
image

## Testing Steps / QA Criteria

- Open deployment link or run branch `ap-et-styling` locally
- Ensure local storage is cleared
- Without clicking on a list name, navigate to both the List and Manage List pages and verify that they look like the screenshots
    - Verify that the 'select a list' button still takes user to the Home page
- Click on a list you know is empty, navigate to the List page and verify that it looks like the screenshot
- Click on a list with items, navigate to the List and Manage List pages and verify that they look like the screenshots
- Verify that all the functionality (e.g. adding an item, purchasing an item, deleting an item, sharing a list, deleting a list) still works as expected

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -5,19 +5,31 @@ const defaultTheme = require('tailwindcss/defaultTheme');
...
 	content: ['./src/**/*.{js,jsx,ts,tsx}'],
 	theme: {
-		colors: {
-			eggshell: '#FDFDF2',
-			sage: '#BBC893',
-			brown: '#D4A373',
-			yellow: '#FEFAE0',
-			vanilla: '#FAEDCD',
-			'pale-green': '#E9EDC9',
+		screens: {
```
Comment (sdmisra) [inline tailwind.config.js:8]: praise: thank you for adding these in, we can use them in the home page as well as navbar to fine tune things, too

**Review by sdmisra — APPROVED** [review]
Praise: These changes look great, I noticed a lot of nice details you all added in your grids with padding and margins that really reformatted the list items into a much better situation for the user.

non-blocking concern for the future: As we merge these remaining PRs prior to our demo, we might need to adjust everything to make it look cohesive, then we'll need to do a bit of accessibility review / check the contrast like Andrea did a little bit ago.

**Review by DevinaG007 — APPROVED** [review]
Praise: It looks pehnomal! You guys clearly put a lot of effort into this, it shows! Our app is looking coherent.

Nit: The background color for the input in the ManageList to share a list is pale-green, but the input boxes in the List view are not, I think we should make all of our forms the same if we can. Personally I think the white background looks cleaner. Shane and I set ours to eggshell for now (because not setting a background color will have it default to white in light mode and black in dark mode, which looked funny.) We could change all of them to a basic white?

**Comment by andiedoescode** [comment]
> Praise: It looks pehnomal! You guys clearly put a lot of effort into this, it shows! Our app is looking coherent.
> 
> Nit: The background color for the input in the ManageList to share a list is pale-green, but the input boxes in the List view are not, I think we should make all of our forms the same if we can. Personally I think the white background looks cleaner. Shane and I set ours to eggshell for now (because not setting a background color will have it default to white in light mode and black in dark mode, which looked funny.) We could change all of them to a basic white?

Oops, yes I meant to change that last form input background color. I can set everything to basic white (#FFF)

**Review by luisaugusto — APPROVED** [review]
Awesome, this is looking great team!

**Pull request merged**